Address John's IESG review of the architecture document (#493)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-architecture.md

This section covers some expected deployment models and their corresponding
security and privacy considerations. Each deployment model is described in
terms of the trust relationships and communication patterns between Client,
Attester, Issuer, and Origin. Entities encapsulated by a bounding box are
assumed to be operated by the same party and therefore collude. Collusion
can enable linking of attestation, issuance, and redemption contexts.

> Entities that are not encapsualted by a bounding box are assumed to not

Author: ```suggestion
Entities that are not encapsulated by a bounding box are assumed to not
```

> Attester, Issuer, and Origin. Entities encapsulated by a bounding box are
> assumed to be operated by the same party and therefore collude. Collusion
> can enable linking of attestation, issuance, and redemption contexts.
> Entities that are not encapsulated by a bounding box are assumed to not
> collude, meaning that entities operated by separate parties do not collude.
> Mechanisms for enforcing non-collusion are out of scope for this architecture.

Reviewer: ```suggestion
Attester, Issuer, and Origin. Entities drawn within the same bounding box are
assumed to be operated by the same party and are therefore able to collude.
Collusion can enable linking of attestation, issuance, and redemption contexts.
Entities not drawn within the same bounding box are assumed to not
collude, meaning that entities operated by separate parties that do not collude.
Mechanisms for enforcing non-collusion are out of scope for this architecture.
```
